schemas/v3.0/schema.yaml

```diff
@@ -0,0 +1,1400 @@
+type: object
+required:
+  - openapi
+  - info
+  - paths
+properties:
+  openapi:
+    type: string
+    enum:
+      - 3.0.0
```

Comment:
Yeah, this should actually change to a pattern that matches anything starting with 3.0 as according to semver, any patch level changes should not affect validation. That said, I'm embarrassed to say I've avoided learning regex as much as I can 😁